go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/go-openapi/spec v0.19.5
 	github.com/golang/mock v1.6.0
 	github.com/jedib0t/go-pretty/v6 v6.3.5
+	github.com/kelseyhightower/envconfig v1.4.0
```

Comment:
This was last released in 2019, so I'm wondering if this project is still maintained. This was also asked in https://github.com/kelseyhightower/envconfig/issues/204

Based on the number of open issues and PRs, do you think it would make sense to look for alternative packages that may be more maintained? Looks like [github.com/sethvargo/go-envconfig](https://github.com/sethvargo/go-envconfig) (recommended in https://github.com/kelseyhightower/envconfig/issues/204#issuecomment-1173043555) has a similar approach and has more activity..